### PR TITLE
Added Owner on a secret "level"

### PR DESCRIPTION
**Pull request by Gaardsholt**
If possible, please test it - it works on my machine at least

### CODE REVIEW

**Comment by Gaardsholt** [comment]
Example of a secret with owner can be seen here: https://github.com/BESTSELLER/harpocrates/pull/38/files#diff-5dfd9fa11d1312184af34c96f4272c57ab6013efd36406f7b9d58da4859752e8

File: files/files.go [hunk 1]
```diff
@@ -48,17 +48,28 @@ func Write(output string, fileName string, content string) {
...
 
 	// set permissions on file and folder
+	fmt.Println("owner", owner)
+	if owner != nil {
+		fmt.Println("owner is set")
+		setPermissions(f, path, output, *owner)
+		return
+	}
 	if config.Config.Owner != -1 {
+		fmt.Println("config.Config.Owner is set")
```
Comment (brondum) [inline files/files.go:58]: i was about to say that this should not be there.
But i guess it is okay to have some output ?

**Review by brondum — APPROVED** [review]
LGTM

**Comment by brondum** [comment]
Please wait with the merge, just remembered a couple of tests :)

**Comment by brondum** [comment]
Works like a charm!

File: files/files.go (hunk 1 above)
Comment (Gaardsholt) [reply files/files.go:58]: naah you are right - that was jus there to debug :D

**Comment by Gaardsholt** [comment]
> Please wait with the merge, just remembered a couple of tests :)

Just let me know when I can merge.

**Comment by brondum** [comment]
> > Please wait with the merge, just remembered a couple of tests :)
> 
> Just let me know when I can merge.

Works like a charm, so merge away :)

**Pull request merged**